DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/DiagnosticContentViewModelTests.cs

```diff
@@ -0,0 +1,240 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Microsoft.Extensions.Configuration;
+    using NUnit.Framework;
+
+    public class DiagnosticContentViewModelTests
+    {
+        private IConfiguration config;
+        private List<int> SelectedTutorials;
+        private const string BaseUrl = "https://example.com";
+        private const int CustomisationId = 5;
+        private const int CentreId = 5;
+        private const int SectionId = 5;
+        private const int CandidateId = 5;
+        private const int ProgressId = 5;
+
+        [SetUp]
+        public void SetUp()
+        {
+            config = A.Fake<IConfiguration>();
+            A.CallTo(() => config["CurrentSystemBaseUrl"]).Returns(BaseUrl);
+            SelectedTutorials = new List<int>();
```

Comment:
I'm not sure this should be a class variable, as it's not used for every test. I would either:
* Inline it and define it for each test
* Define it as a constant named `EmptySelectedTutorials` to make it clear how it's meant to be used. 

Does that make sense? 🙂